lib/DDG/Spice/SalesTaxHoliday.pm

```diff
@@ -0,0 +1,36 @@
+package DDG::Spice::SalesTaxHoliday;
+#ABSTRACT: Returns the sales tax 'holiday' detailed information for any state in the United States.
+
+use strict;
+use DDG::Spice;
+use Locale::SubCountry;
+triggers any => 'holiday sales tax', 'sales tax holiday', 'holiday sales tax in', 'holiday sales tax for', 'sales tax holiday in', 'sales tax holiday for';
+spice to => 'http://dev.snapcx.io:9100/taxv1/getTaxHolidayInfo?request_id=DDG_Requestor&state=$1';
+spice wrap_jsonp_callback => 1;
+#Create US SubCountry object
+my $US = new Locale::SubCountry("US");
+
+# Handle statement
+handle remainder_lc => sub {
+    my ($stateCode); #Define vars
+    s/^what is (the)?//g; # strip common words
+    return '' unless $_; # Guard against "no answer"
```

Comment:
Hi @moollaza , I thought about it. In this API, if there is empty value passed to API parameter, then also it returns sales tax holiday for ALL states. If we just do return, then API won't get invoked.